src/algebra/CommRing/colimits.lean

```diff
@@ -60,7 +60,8 @@ inductive relation : prequotient F → prequotient F → Prop
 | symm : Π (x y) (h : relation x y), relation y x
 | trans : Π (x y z) (h : relation x y) (k : relation y z), relation x z
 -- There's always a `map` relation
-| map : Π (j j' : J) (f : j ⟶ j') (x : (F.obj j).α), relation (of j' ((F.map f) x)) (of j x)
+| map : Π (j j' : J) (f : j ⟶ j') (x : (F.obj j).α),
+      relation (of j' ((@coe_fn _ ring_hom.has_coe_to_fun $ F.map f) x)) (of j x)
```

Comment:
Oof, I see. This is bad news. I will investigate.

Reply:
Sorry, it should work now. For some reason, it didn't work with `hom` being a field of `bundled_hom`, not an argument.

Reply:
Actually, this compilation failure was the reason for me to move `hom` to the argument.

Reply:
Marking as resolved.